src/libs/ajax/workspaces/providers/ExportWorkflowToWorkspaceProvider.ts

```diff
@@ -40,3 +40,33 @@ export const makeExportWorkflowFromWorkspaceProvider = (
     },
   };
 };
+
+/**
+ * Create a provider to export a workflow from the Broad Methods Repository to a
+ * workspace, with a blank configuration.
+ *
+ * @param {MethodRepoMethod} sourceMethod - the method to be exported from the
+ * Methods Repository. The methodNamespace, methodName, and methodVersion
+ * properties should be present.
+ */
+export const makeExportWorkflowFromMethodsRepoProvider = (
+  sourceMethod: MethodRepoMethod
+): ExportWorkflowToWorkspaceProvider => {
+  return {
+    export: async (destWorkspace: WorkspaceInfo, destWorkflowName: string, options: AbortOption = {}) => {
+      const { signal } = options;
+
+      // Remove placeholder root entity type from template before importing -
+      // the user can select their own on the workflow configuration page
+      const { rootEntityType, ...template } = await Ajax(signal).Methods.template(sourceMethod);
```

Comment:
`rootEntityType` appears to be unused. It looks like you can use the following slightly strange syntax to skip destructuring it (did not try running).
```suggestion
      const {, ...template } = await Ajax(signal).Methods.template(sourceMethod);
```

Reply:
This syntax unfortunately does not work. I am intentionally removing `rootEntityType` from `template`, and since the object returned by the Ajax call has multiple fields, JavaScript needs to know which one I want to discard.